build.gradle

```diff
@@ -104,11 +104,13 @@ tasks.register('testLab') {
             executable 'gcloud'
             args('beta', 'firebase', 'test', 'android', 'run',
                     '--type', 'instrumentation',
-                    '--num-uniform-shards=50',
+                    '--num-uniform-shards=25',
                     '--app', 'collect_app/build/outputs/apk/debug/ODK-Collect-debug.apk',
                     '--test', 'collect_app/build/outputs/apk/androidTest/debug/ODK-Collect-debug-androidTest.apk',
-                    '--device', 'model=MediumPhone.arm,version=30,locale=en,orientation=portrait',
-                    '--timeout', '10m'
+                    '--device', 'model=MediumPhone.arm,version=34,locale=en,orientation=portrait',
+                    '--timeout', '10m',
+                    '--directories-to-pull', '/sdcard',
```

Comment:
The commit title says `Make view hierarchy available on failures`. Could you elaborate?

Reply:
Good point! I've updated the commit message (https://github.com/getodk/collect/pull/6059/commits/8ea019fbc6933d47b968a4c4700bcea8d27ecdd4) to make it clearer.